src/state-summary/state-card-climate.html

```diff
@@ -52,14 +56,23 @@
       type: Boolean,
       value: false,
     },
-
     stateObj: {
       type: Object,
     },
   },
 
   computeTargetTemperature: function (stateObj) {
-    return stateObj.attributes.temperature + ' ' + stateObj.attributes.unit_of_measurement;
+    if (stateObj.attributes.operation_mode.includes('Auto')) {
```

Comment:
It is a string, includes comment still is valid. Don't use it.

Also, instead of testing for a specific value is included, just use string comparison: `operation_mode === 'Auto'`.